app/screens/AddExpectingChildProfile.tsx

```diff
@@ -127,7 +129,7 @@ const AddExpectingChildProfile = ({ route, navigation }: Props): any => {
     const insertData: any = editScreen ? await getNewChild(childData?.uuid, "true", null, '', plannedTermDate, name, '', '',childData?.createdAt) : await getNewChild('', "true", null, '', plannedTermDate, name, '', '',null);
     const childSet: Array<any> = [];
     childSet.push(insertData);
-    addChild(languageCode, editScreen, 2, childSet, dispatch, navigation, childAge, null,null);
+    addChild(languageCode, editScreen, 2, childSet, dispatch, navigation, childAge, null,null,netInfoval);
```

Comment:
Let's please use netInfo everywhere instead of netInfoval. 
I see formatting needs to be fixed for some of your changes. Can you please double check?

Reply:
okay sure, I will rename netInfoval  to netInfo everywhere.